### PR TITLE
Fixed collateral input problem

### PR DESCRIPTION
**Pull request by Polybius93**

### CODE REVIEW

File: src/modals/DepositModal.js [hunk 1]
```diff
@@ -76,8 +76,8 @@ export default function DepositModal({ isOpen, closeModal, walletType }) {
   };
 
   const createLoanContract = () => ({
-    vaultLoanAmount: fixedTwoDecimalUnshift(vaultLoanAmount),
-    BTCDeposit: customShiftValue(collateralAmount, 8, false),
+    vaultLoanAmount: parseInt(fixedTwoDecimalUnshift(vaultLoanAmount)),
+    BTCDeposit: parseInt(customShiftValue(collateralAmount, 8, false)),
```
Comment (scolear) [inline src/modals/DepositModal.js:80]: this didn't work until now? or why this change?

Comment (Polybius93) [reply src/modals/DepositModal.js:80]: It did work, but with 0.55 it didn't arrive as an integer, probably because of a floating point error

Comment (scolear) [reply src/modals/DepositModal.js:80]: Oh right, gotcha. But both of these functions return a number anyway, not a string, so are you sure this fixes this?

Comment (Polybius93) [reply src/modals/DepositModal.js:80]: Well it did fix it
